vendor.conf

```diff
@@ -143,7 +143,7 @@ github.com/inconshreveable/mousetrap 76626ae9c91c4f2a10f34cad8ce83ea42c93bb75
 github.com/Nvveen/Gotty a8b993ba6abdb0e0c12b0125c603323a71c7790c https://github.com/ijc25/Gotty
 
 # docker CLI
-github.com/docker/cli 8d84dffd5547772e1f110e9e735c13af262d4ee5 https://github.com/resin-os/balena-cli
+github.com/docker/cli ce80489d66dfc63ac005f55f5bdca2f03abee6e9 https://github.com/resin-os/balena-cli
```

Comment:
Updated commit hash for the balena-cli commit that does the balenaEngine renaming